llvm/include/llvm/Transforms/Utils/BuildLibCalls.h

```diff
@@ -247,7 +247,7 @@ namespace llvm {
 
   /// Emit a call to the malloc function.
   Value *emitMalloc(Value *Num, IRBuilderBase &B, const DataLayout &DL,
-                    const TargetLibraryInfo *TLI);
+                    const TargetLibraryInfo *TLI, unsigned AddrSpace);
```

Comment:
I think it might be simpler overall if all of these directly passed a pointer type rather than only the address space?

Reply:
Done